src/overflow/setup_bands.py

```diff
@@ -0,0 +1,32 @@
+from osgeo import gdal
+from contextlib import contextmanager
+
+
+gdal.UseExceptions()
+
+
+@contextmanager
+def setup_bands(input_path, output_path):
```

Comment:
util/raster might be a better place for this instead of having it in it's own file? If that file is getting large we can split into modules under util/